.gitignore

```diff
@@ -31,3 +31,6 @@ docs/
 *.lst
 
 # End of https://www.gitignore.io/api/d
+
+# Config environment file
+/.env
```

Comment:
`.env` is commonly used to store credentials and other sensitive information. As the heroku deployment is configured with environment variables, a `.env` is very handy to test the daily cron feature locally.